etc/ipsec.conf

```diff
@@ -16,9 +16,9 @@ conn %default
 conn rw
     # http://wiki.loopop.net/doku.php?id=server:vpn:strongswanonopenvz
     # https://wiki.strongswan.org/projects/strongswan/wiki/ForwardingAndSplitTunneling
-    leftsubnet=0.0.0.0/0
+    leftsubnet=0.0.0.0/0,::/0
     # end ref
     leftfirewall=yes
     right=%any
-    rightsourceip=10.8.0.0/16
+    rightsourceip=10.8.0.0/16,2a00:1450:400c:c05::/64
```

Comment:
Oops. Yes you are right it should. However I don't think it should come out of the IPv6 mapped addresses, because it should probably be a full /64 to enable people to do SLAAC at some point. I think it should come out of be a /64 out of the `fc00::/7`subnet. I'll update this later today.